app/routes/reductions.js

```diff
@@ -0,0 +1,76 @@
+const organisationUnitConstants = require('../constants/organisation-unit')
+const reductionsService = require('../services/reductions-service')
+const Reduction = require('../services/domain/new-reduction')
+
+module.exports = function (router) {
+  router.get('/:organisationLevel/:id/reductions', function (req, res, next) {
+    var organisationLevel = req.params.organisationLevel
+    var id = req.params.id
+    var successText = req.body.successText ? req.params.successText : null
+
+    if (organisationLevel !== organisationUnitConstants.OFFENDER_MANAGER.name) {
+      throw new Error('Only available for offender manager')
+    }
+
+    var reductionsPromise = reductionsService.getReductions(id)
+
+    return reductionsPromise.then(function (result) {
+      return res.render('reductions', {
+        breadcrumbs: result.breadcrumbs,
+        linkId: id,
+        title: result.title,
+        active: [],
+        scheduled: [],
+        archived: [],
```

Comment:
Are these three needed for this story? Can they be removed and this story still get closed as we aren't displaying?

Reply:
I would leave them for now, as they are place holders and we can expect the structure for the upcoming story.